test/deploy_header.js

```diff
@@ -0,0 +1,80 @@
+/* globals describe,it */
+const assert = require('assert');
```

Comment:
consider also 'chai' or similar if built-in asserts seem awkward

Reply:
We just need this for the rejects method